pkg/state/appender.go

```diff
@@ -790,6 +791,87 @@ func (a *txAppender) appendFixSnapshots(fixSnapshots []proto.AtomicSnapshot, blo
 	return nil
 }
 
+func (a *txAppender) isLastBlockFinalized(
```

Comment:
Strange naming, is it better to name the function "updateFinalization" or "calculateFinalization"?